src/content/plugins/internal-plugins.md

```diff
@@ -64,12 +63,6 @@ Plugins, which add entry chunks to the compilation.
 
 Adds an entry chunk on compilation. The chunk is named `chunkName` and contains only one module (plus dependencies). The module is resolved from `request` in `context` (absolute path).
 
-### MultiEntryPlugin
```

Comment:
It's removed in this commit https://github.com/webpack/webpack/commit/6d5310fbd609f60ce36dc3611dbe9a76f61a0848